internal/database/db.go

```diff
@@ -171,6 +171,21 @@ func (dbs *SQLdb) getHeader(fileID int) ([]byte, error) {
 	return header, nil
 }
 
+// GetHeaderStableId retrieves the file header by using stable id
+func (dbs *SQLdb) GetHeaderStableId(stableID string) (string, error) {
```

Comment:
```suggestion
func (dbs *SQLdb) GetHeaderForStableId(stableID string) (string, error) {
```
slightly more descriptive name